plugins/nanospell/plugin.js

```diff
@@ -564,13 +580,17 @@
 				return editor.getSelection().getSelectedText().length == 0;
 			}
 
-			var spellTicker = null;
+			function startSpellCheckTimer(delay) {
+				if (self._timer) {
```

Comment:
Why not `if (self._timer === null) {` (or whatever the preferred null-checking is), rather than an empty clause?